docs/src/frontend/index.md

```diff
@@ -666,6 +666,15 @@ acc.tokenAccepted(token) => Promise<boolean>
 Returns a Promise that returns if an account may accept a given token.
 This does nothing on some consensus networks, but should always be used to ensure your frontend is blockchain agnostic.
 
+---
+@{ref("js", "acc.tokensAccepted")}
+```js
+acc.tokensAccepted() => Promise<Array<Token>>
```

Comment:
Oh! I thought of two things... add the `stdlib.tokensAccept(acc)` version and add both to the docs; almost forgot!